src/test/java/org/prebid/server/privacy/gdpr/tcfstrategies/purpose/typestrategies/FullEnforcePurposeStrategyTest.java

```diff
@@ -1039,4 +1039,22 @@ public void shouldReturnExcludedVendors() {
         // then
         assertThat(result).usingFieldByFieldElementComparator().containsOnly(vendorPermission2);
     }
+
+    @Test
+    public void shouldReturnExcludeInactiveBidders() {
+        // given
+        // a vendorId is null when the bidder is not enabled, thus inactive
+        final VendorPermission vendorPermission = VendorPermission.of(
+                null, "inactive bidder", PrivacyEnforcementAction.restrictAll());
+        final VendorPermissionWithGvl vendorPermissionWithGvlNoVendorId = VendorPermissionWithGvl.of(
+                vendorPermission, VendorV2.empty(null)
+        );
+
+        // when
+        final Collection<VendorPermission> result = target.allowedByTypeStrategy(PURPOSE, tcString,
+                singleton(vendorPermissionWithGvlNoVendorId), emptyList(), true);
+
+        // then
+        assertThat(result).isEmpty();
+    }
```

Comment:
```
    @Test
    public void shouldReturnOnlyExcludedVendorWhenVendorIdsIsNotDefined() {
        // given
        // a vendorId is null when the bidder is not enabled
        final VendorPermission vendorPermission1 = VendorPermission.of(null, "active bidder",
                PrivacyEnforcementAction.restrictAll());
        final VendorPermission vendorPermission2 = VendorPermission.of(null, "inactive bidder",
                PrivacyEnforcementAction.restrictAll());
        final VendorPermissionWithGvl vendorPermissionWitGvl1 = VendorPermissionWithGvl.of(vendorPermission1,
                VendorV2.empty(null));
        final VendorPermissionWithGvl vendorPermissionWitGvl2 = VendorPermissionWithGvl.of(vendorPermission2,
                VendorV2.empty(null));

        // when
        final Collection<VendorPermission> result = target.allowedByTypeStrategy(PURPOSE, tcString,
                singleton(vendorPermissionWitGvl1), singleton(vendorPermissionWitGvl2), true);

        // then
        assertThat(result).usingFieldByFieldElementComparator().containsOnly(vendorPermission2);

        verifyZeroInteractions(allowedVendors);
        verifyZeroInteractions(purposesConsent);
    }
```

Lets add some verification that we don't triger any logic and just remove empty vendor ids, but excluded is still fine